src/models/voting/VotingMainModel.js

```diff
@@ -0,0 +1,31 @@
+import Immutable from 'immutable'
+import { abstractFetchingModel } from 'models/AbstractFetchingModel'
+import BigNumber from 'bignumber.js'
+
+export default class VotingMainModel extends abstractFetchingModel({
+  voteLimitInTIME: null,
+  list: new Immutable.Map(),
+  pollsCount: new BigNumber(0),
+  activePollsCount: new BigNumber(0),
+  lastPoll: new BigNumber(0),
+}) {
+  pollsCount (value) {
+    return this._getSet('pollsCount', value)
+  }
+
+  list (value) {
```

Comment:
use abstractFetchingCollection instead

Reply:
it's list of polls. list = new VotingCollection(). It's ok.